test/acceptance/behave/components/ngsiv2/entities/list_all_entities.feature

```diff
@@ -0,0 +1,795 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+
+Feature: list all entities with get requests in NGSI v2. "GET" - /v2/entities/
+  As a context broker user
+  I would like to list all entities with get requests in NGSI v2
+  So that I can manage and use them in my scripts
+
+  BackgroundFeature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file and restart service
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  @happy_path
+  Scenario:  list all entities in NGSI v2
+    Given  a definition of headers
+      | parameter          | value                |
+      | Fiware-Service     | test_list_happy_path |
+      | Fiware-ServicePath | /test                |
+      | Content-Type       | application/json     |
+    And create "5" entities with "2" attributes
+      | parameter        | value                   |
+      | entities_type    | room                    |
+      | entities_id      | room2                   |
+      | attributes_name  | timestamp               |
+      | attributes_value | 017-06-17T07:21:24.238Z |
+      | attributes_type  | date                    |
+      | metadatas_number | 2                       |
+      | metadatas_name   | very_hot                |
+      | metadatas_type   | alarm                   |
+      | metadatas_value  | random=10               |
+    And verify that receive several "Created" http code
+    When get all entities
+      | parameter | value |
+      | limit     | 3     |
+      | offset    | 2     |
+    Then verify that receive an "OK" http code
+    And verify that all entities are returned
```

Comment:
That it means that the info returned info is checked to match with the sent requests or just the amount of entities returned?